addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/json/request/ICloudFindMyDeviceRequest.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.icloud.internal.json.request;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Serializable class to create a "Find My Device" json request string.
+ *
+ * @author Patrik Gfeller
+ *
+ */
+public class ICloudFindMyDeviceRequest {
+    @SerializedName("device")
+    String deviceId;
+    final String subject = "Find My Device alert";
```

Comment:
What is this? Some constant then make it one? Should it be private / public or like now package-private

Reply:
> What is this? Some constant then make it one? Should it be private / public or like now package-private